eb7_sls_helper/src/gh_action_interface.py

```diff
@@ -308,15 +328,26 @@ def run_tox(
     sls = discover_file(changes_list, args["filename"])
     log.info(f"Discovered: {' '.join(sls)}")
 
+    serverless_config = read_serverless_config(' '.join(sls))
```

Comment:
you can remove the join as it is not needed 
```suggestion
    serverless_config = read_serverless_config(sls)
```

Reply:
we return list, not string